ArduCopter/mode_zigzag.cpp

```diff
@@ -118,12 +117,16 @@ void Copter::ModeZigZag::save_or_move_to_destination(uint8_t dest_num)
 }
 
 // return manual control to the pilot
-void Copter::ModeZigZag::return_to_manual_control()
+void Copter::ModeZigZag::return_to_manual_control(bool maintain_target)
```

Comment:
This new argument is unused.

Reply:
txs, removed

Reply:
oops, actually it's required but we were missing a check, fixed!